deploy/ocp/manifests/0.7.4/0000_30_13-packageserver.yaml

```diff
@@ -0,0 +1,151 @@
+##---
+# Source: olm/templates/0000_30_13-packageserver.yaml
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.packages.apps.redhat.com
+spec:
+  caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM5VENDQWQyZ0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFjTVJvd0dBWURWUVFERXhGd1lXTnIKWVdkbExYTmxjblpsY2kxallUQWVGdzB4T0RFd01Ua3hPRFEyTlRaYUZ3MHlPREV3TVRZeE9EUTJOVFphTUJ3eApHakFZQmdOVkJBTVRFWEJoWTJ0aFoyVXRjMlZ5ZG1WeUxXTmhNSUlCSWpBTkJna3Foa2lHOXcwQkFRRUZBQU9DCkFROEFNSUlCQ2dLQ0FRRUF3SmQ5Yk5mdzliMW1PczRncDdlZ0FWKzlFNEgvVUhReDBZSzhoMXFvb2c0eTZWWjIKMVdkUDhLaW5EalJVK0tUamxOazQ4b0Z1aXFKU2VNTmRYS2Y4bUEvK3F6Q2ZFRWt3OEdNMnZLSVB2NkRQeUY0NApBWGhjbVJjMzlFQ0oyU2d0TmxGRU5senhiWFdjQ1ZHRnk4VVJGaVpqdFNJdEpTRTFjVWNnT0dYWGFhVE5ib2Z0Cklvd3FtK0YxWHFEQVhCM1h2c2M2bUkwM0ovZjIvU1AxNC91TVo5SmlaSXg2OCs5OFdPRDhHdmpibWhuSzk5azcKWHdzMm1ZQm04dVR6REZQSjJNd3BPb2xrTEg5ekpjVU9zMVJuV2hvYm5Fek05MGt0NUVPSEdseW9KdGhZTHRWWgovWVlMN2t5YjF4TjlYZ2RKalVOazUwcFhLYlE5RGpQWGczMmwzd0lEQVFBQm8wSXdRREFPQmdOVkhROEJBZjhFCkJBTUNBcVF3SFFZRFZSMGxCQll3RkFZSUt3WUJCUVVIQXdFR0NDc0dBUVVGQndNQ01BOEdBMVVkRXdFQi93UUYKTUFNQkFmOHdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBQnlWVjk5Ris1K0pxcmVoN3VFTjlRMDNZd1QrUnR1MgowYzJqQ0g4REltSHdUVlJhUTNxdzJwd1NUV0VJaEpGT2wvOG1tdkl2VHdoNmM1OGxrNVhYeDl4cHVQc0dtTWtjClFZWkFIZVRSeElWS08renFLUHdSZmdXRzV3WTB6UGNZZUFadU5mbVlzR2NmVHNGK2JFc2FsMXY0S0laQkFNU3QKUlFDaW5jWCtpT1BTK0dEMUtMSGJWUEQrM0RkUUtVeC85cURJSHk2VERCUTJtaVJmN2hFblNWM2dPWlVmWWFvTQptazVuMTRjOG9aMFVOUkVZZHluVmNBL2ZnbGNkcnR2UFp5eE9Ca0tadWxINGhJVDhQTVprcTBuYmxTRlh2anJTCjUwVTlVZ3hsc2tOc3kvbGtLNkJhNFozUEE0d2NPSVphd1psUmtWMmRvcyt3OHVGbnI3Q1BWQlE9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
It would be better if you add the annotation `'service.alpha.openshift.io/inject-cabundle=true'` and let the service-erving-cert-signer-operator inject the caBundle for you: https://github.com/openshift/service-serving-cert-signer/pull/35/files#diff-04c6e90faac2675aa89e2176d2eec7d8R65

Reply:
My only concern is how this plays out with upstream deployments. Would our manifests ensure the service-serving-cert-signer is instantiated before the package-server? (I understand that with OpenShift and the installer we can ensure existence and ordering)